templates/meta_review.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```

Comment:
Can you post a screenshot of the generated screen?

Reply:
There's no 'UI' for that. The template is modified from templates/contributors.html, which looks like http://coala-community.netlify.com/contributors/